### PR TITLE
Update project-service-core-solution-version-history.md

### PR DESCRIPTION
**Pull request by stsporen**
Updated release notes.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 1]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
   
 |**Version**|**Release date**|**Update description**|
 |:-----|:-----|:-----|
-|1.0.4.71 <br/> |November 7, 2020 <br/> | Items resolved in this version: 
+|1.0.6.154 | January 12, 2021 |Items resolved in this version:|
+|  |  | - Bug: Breaking change by Renaming webresource |
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:35]:
```suggestion
|  |  | - Bug: Breaking change by renaming web resource |
```

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 2]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
...
+|  |  | - Bug: Update webresources in base solution |
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:36]:
```suggestion
|  |  | - Bug: Update web resources in base solution |
```

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 3]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
...
+|  |  | - Bug: Named orgs need default PostImport configuation parameters |
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:37]:
```suggestion
|  |  | - Bug: Named orgs need default PostImport configuration parameters |
```

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 4]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
...
+|  |  | - Bug: Rename Common Data Service User to Project Common |
+|  |  | - Bug: Named orgs toolbar updates |
+|  |  | - Feature: Add customizable entities to app module |
+|  |  | Additional improvements in performance and bug fixes |
+|1.0.4.71  | November 7, 2020 | Items resolved in this version: |
 |  |  | - Bug: Manual deployment fails with post-import error|
 |  |  | - Bug: User timezone related notification is not localised|
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:44]:
```suggestion
|  |  | - Bug: User timezone related notification is not localized |
```

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 5]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
...
 |  |  | - Feature: Added support for Project Operations|
 |  |  | Additional improvements in performance and bug fixes |
-|1.0.1.605  <br/> |April 3, 2020  <br/> |Improvements in installation of the solution to address instances where the default team has been modified. Additional bug fixes and performance improvements. <br/> |
-|1.0.1.427  <br/> |January 2020  <br/> |Several bug fixes and performance improvements.  <br/> |
-|1.0.1.211  <br/> |November 2019  <br/> |Several bug fixes and performance improvements.  <br/> |
-
-
-   
-
+|1.0.1.605 | April 3, 2020  |Improvements in installation of the solution to address instances where the default team has been modified. Additional bug fixes and performance improvements. |
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:49]:
```suggestion
|1.0.1.605 | April 3, 2020  |Improvements in installation of the solution to address instances where the default team has been modified. Additional bug fixes and performance improvements |
```

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 6]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
...
+|1.0.1.427 | January 2020  |Several bug fixes and performance improvements.  |
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:50]:
```suggestion
|1.0.1.427 | January 2020  |Several bug fixes and performance improvements  |
```

File: ProjectWeb/project-service-core-solution-version-history.md [hunk 7]
```diff
@@ -31,18 +31,22 @@ If you want to upgrade your solution within Dynamics 365 when a newer version is
...
+|1.0.1.211 | November 2019  |Several bug fixes and performance improvements.  |
```
Comment (JohanFreelancer9) [inline ProjectWeb/project-service-core-solution-version-history.md:51]:
```suggestion
|1.0.1.211 | November 2019  |Several bug fixes and performance improvements  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**